channels/us.m3u

```diff
@@ -6586,7 +6588,7 @@ https://wxeldt.lls.cdn.pbs.org/wxeldt/6f553836-7e24-445a-8c3c-cc418aaba9dc/prima
 https://wfsuhd.lls.cdn.pbs.org/wfsuhd/da7db8ae-5c3b-450e-9a25-76aa661eac79/primary.m3u8
 #EXTINF:-1 tvg-id="WEDU.us" tvg-name="WEDU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="General",PBS Tampa (WEDU) [Geo-blocked]
 https://wedudt.lls.cdn.pbs.org/wedudt/c4ab80c1-ca8c-4a6e-a8b5-c5995919792b/primary.m3u8
-#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="General",PBS [Geo-blocked]rgia (WVAN) (WXGA) [Geo-blocked]
+#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="General",PBS Georgia (WVAN) (WXGA) [Geo-blocked]
```

Comment:
WVAN-TV Savannah, Georgia

WXGA isn't in this stream.

Reply:
Both stations are 135km away from each other. Broadcasted content is the same I guess

Reply:
Sounds fair to me.

Reply:
Yes PBS Georgia is a statewide network instead of the affiliate in Atlanta so I took their two top stations and put it in.